pkg/handlers/archive.go

```diff
@@ -161,9 +192,11 @@ func (a *Archive) extractorHandler(archiveChan chan []byte) func(context.Context
 		}
 		defer fReader.Close()
 
-		if common.SkipFile(f.Name()) {
-			lCtx.Logger().V(5).Info("skipping file", "filename", f.Name())
-			return nil
+		if a.skipBinaries {
+			if common.SkipFile(f.Name()) {
+				lCtx.Logger().V(5).Info("skipping file", "filename", f.Name())
+				return nil
+			}
 		}
```

Comment:
Same question here, why was this changed to only skip files if `skipBinaries` is set?

Reply:
updated